t/unit/utils/test_json.py

```diff
@@ -29,19 +28,18 @@ class test_JSONEncoder:
     def test_datetime(self):
         now = datetime.utcnow()
         now_utc = now.replace(tzinfo=pytz.utc)
-        serialized = loads(dumps({
+
+        original = {
             'datetime': now,
             'tz': now_utc,
             'date': now.date(),
-            'time': now.time()},
-        ))
-        assert serialized == {
-            'datetime': now,
-            'tz': now_utc,
-            'time': now.time().isoformat(),
-            'date':  date(now.year, now.month, now.day),
+            'time': now.time(),
         }
 
+        serialized = loads(dumps(original))
+
+        assert serialized == original
```

Comment:
I improved this test, now it exactly matches original (i added `time` type support).